lib/jsonapi-response.js

```diff
@@ -10,7 +10,7 @@ module.exports = (data, config) => {
   response.data.id = TypeMapping.toExternal(data._id);
   response.data.attributes = getAttributes(data._source);
 
-  response.relationships = getRelationships(data._source);
+  response.relationships = getRelationships(data._source, response, config);
```

Comment:
You're already mentioned this but relationships should be on the `data` property.